src/moduler/aktivitet/visning/avtalt-container/ForhaandsorienteringBrukerVisning.tsx

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 
 import { Aktivitet } from '../../../../datatypes/aktivitetTypes';
+import { markerForhaandsorienteringSomLest } from '../../aktivitet-actions';
+import { markerForhaandsorienteringSomLestArenaAktivitet } from '../../arena-aktiviteter-reducer';
 import DeleLinje from '../delelinje/delelinje';
-import Forhaandsorenteringsvisning from '../hjelpekomponenter/Forhaandsorenteringsvisning';
+import Forhaandsorienteringsvisning from '../hjelpekomponenter/forhaandsorientering/Forhaandsorienteringsvisning';
 
 interface Props {
     aktivitet: Aktivitet;
 }
 
 const ForhaandsorienteringBrukerVisning = (props: Props) => {
     const { aktivitet } = props;
-    const forhaandsorienteringTekst = aktivitet.forhaandsorientering?.tekst;
+
+    const dispatch = useDispatch();
+
+    const onMarkerSomLest = () => {
+        if (aktivitet.arenaAktivitet) {
+            dispatch(markerForhaandsorienteringSomLestArenaAktivitet(aktivitet));
+        } else {
+            dispatch(markerForhaandsorienteringSomLest(aktivitet));
+        }
+    };
 
     return (
         <>
             <DeleLinje />
-            <Forhaandsorenteringsvisning
-                forhaandsorienteringTekst={forhaandsorienteringTekst}
-                hidden={!forhaandsorienteringTekst}
+            <Forhaandsorienteringsvisning
+                forhaandsorientering={aktivitet.forhaandsorientering}
+                markerSomLest={onMarkerSomLest}
             />
             <DeleLinje />
         </>
```

Comment:
Vi burde kanskje se litt på hvordan ting er satt opp nå fordi dette blir veldig forvirrende. Denne brukervisningen står for visning av FHO for både aktivitet og arenaaktivitet, men den er plassert i avtaltMedNav-mappen 🤔